### ISSUE TITLE
Immersive Portals compatibility?

### ISSUE THREAD

**Issue by Moleculor** [MoreThanHidden/RestrictedPortals#32]
I'm not holding my breath, but is there any chance this mod could ever work with Immersive Portals?

**Comment by MoreThanHidden** [comment]
Does it not already work? I'll do some testing.

**Comment by MoreThanHidden** [comment]
Yeah immersive portals is breaking the Forge EntityTravelToDimensionEvent, so would need them to fix that.
